driver-core/src/main/java/com/datastax/driver/core/QueryOptions.java

```diff
@@ -42,10 +42,32 @@
      */
     public static final boolean DEFAULT_IDEMPOTENCE = false;
 
+    public static final int DEFAULT_MAX_PENDING_REFRESH_NODE_LIST_REQUESTS = 20;
+
+    public static final int DEFAULT_MAX_PENDING_REFRESH_NODE_REQUESTS = 20;
+
+    public static final int DEFAULT_MAX_PENDING_REFRESH_SCHEMA_REQUESTS = 20;
+
+    public static final int DEFAULT_REFRESH_NODE_LIST_INTERVAL_MILLIS = 1000;
+
+    public static final int DEFAULT_REFRESH_NODE_INTERVAL_MILLIS = 1000;
+
+    public static final int DEFAULT_REFRESH_SCHEMA_INTERVAL_MILLIS = 1000;
```

Comment:
Is this default too aggressive?  For example, if doing a schema migration, you may be changing a number of tables/keyspaces at a time.  If doing this responsibility, you'd wait for schema agreement after each change, so you could be looking at ~10 seconds between operation, in which case the debouncing will not be very helpful.